packages/rsu_client.rb

```diff
@@ -0,0 +1,73 @@
+require 'package'
+
+class Rsu_client < Package
+  description 'RSU-Client is a RuneScape Client Launcher written for the now Legacy client and now OldSchool.'
+  homepage 'https://github.com/rsu-client/rsu-client'
+  version '4.3.7'
+  source_url 'https://github.com/rsu-client/rsu-client/archive/v4.3.7.tar.gz'
+  source_sha256 '7cc589de111baad956c0c41a4d7f1b45168886bf623dd3886d07fa3d1dfd604c'
+
+  depends_on 'jdk8'
+  depends_on 'p7zip'
+  depends_on 'perl'
```

Comment:
No need since perl is already a core package.

Reply:
You know, on an unrelated topic, maybe we should stop doing this. I doubt the average user uses the toolchain on a daily basis, and you can't even install a simple text editor without having missing library errors if you don't have `buildessential` installed. Plus, with #3106, dependency loops and speed shouldn't be an issue.

Reply:
Stop doing what?  Let's not get sloppy by being lazy.

Reply:
Yeah, even if we have speed improvements coming up doesn't mean we shouldn't do our best to remove redundancies.
Though I do understand the annoyance of getting a dozen comments on a file just for redundant dependencies 🤣
Perhaps we could avoid immediately pointing them out, and instead come up with a tool to automatically remove them and run it somewhat regularly?

Reply:
Yes, but I'd like to have a working Chromebrew installation without installing `buildessential`. Currently, if you omit installing `buildessential` during the initial install phase, you have to hand-pick out the dependencies for some packages.
Example: If you run a fresh Chromebrew installation and remove any instance of `buildessential` from `install.sh`, and then you run `crew install nano && nano`, you get an error about a missing `libmagic.so` (from `filecmd`).